Tests/RoslynPluginGeneratorTests/SonarQube.Plugins.Roslyn.RoslynPluginGeneratorTests.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.CommandLine" Version="6.3.1" GeneratePathProperty="true" />
+    <PackageReference Include="NuGet.CommandLine" Version="$(NuGetPackagesVersion)" GeneratePathProperty="true" />
```

Comment:
Does this need to be a separate ItemGroup? I'd merge them like the other ones

Reply:
Personally, I'd prefer to separate the other ones out  so they're like this one.
But I don't think it's important enough to do now.